pkg/bindings/network/network.go

```diff
@@ -13,11 +14,24 @@ import (
 
 // Create makes a new CNI network configuration
 func Create(ctx context.Context, network *types.Network) (types.Network, error) {
+	return CreateWithOptions(ctx, network, nil)
+}
+
+func CreateWithOptions(ctx context.Context, network *types.Network, extraCreateOptions *ExtraCreateOptions) (types.Network, error) {
```

Comment:
I think naming it just `CreateOptions` sounds better but I am also ok with this one.

Reply:
The code already has `CreateOptions` struct (this is the reason I named the new struct `ExtraCreateOptions`). So, I think the `With` does help here

Reply:
Oh no, looks like I forgot to remove the old type when I broke the network code with the v4.0 bump.
Make sense in this case.

Reply:
OK, resolving then